channel.yaml

```diff
@@ -0,0 +1,35 @@
+# Example channels config
+channels:
+- name: stable
+  latest: v1.17.4+k3s1
+- name: latest
+  latestRegexp: .*
+  excludeRegexp: ^[^+]+-
+- name: testing
+  latestRegexp: .*
+- name: v1.16
+  latestRegexp: v1\.16\..*
+  excludeRegexp: ^[^+]+-
+- name: v1.16-testing
+  latestRegexp: v1\.16\.*
+- name: v1.17
+  latestRegexp: v1\.17\..*
+  excludeRegexp: ^[^+]+-
+- name: v1.17-testing
+  latestRegexp: v1\.17\..*
+- name: v1.0
+  latestRegexp: v1\.0\..*
+  excludeRegexp: ^[^+]+-
+- name: v1.0-testing
+  latestRegexp: v1\.0\..*
+releases:
```

Comment:
can you drop the releases section all together? It is only needed by the embedded channel-server that runs inside of rancher, should not be needed/used by this public service.